lib_nbgl/src/nbgl_use_case.c

```diff
@@ -1146,28 +1146,36 @@ static void displayFullValuePage(const nbgl_contentTagValue_t *pair)
                                                   .backAndText.token  = 0,
                                                   .backAndText.tuneId = TUNE_TAP_CASUAL,
                                                   .backAndText.text   = PIC(pair->item)};
-    const char              *info;
-    genericContext.modalLayout = nbgl_layoutGet(&layoutDescription);
+    genericContext.modalLayout                 = nbgl_layoutGet(&layoutDescription);
     // add header with the tag part of the pair, to go back
     nbgl_layoutAddHeader(genericContext.modalLayout, &headerDesc);
-    // add full value text
-    if (pair->extension->explanation == NULL) {
+    // add either QR Code or full value text
+    if (pair->extension->aliasType == QR_CODE_ALIAS) {
+#ifdef NBGL_QRCODE
```

Comment:
should we move the #ifdef before the if and #endif between the else and "{" ?

Reply:
Honnestly I prefer not to, because of the else condition... if used without flag, il will simply display nothing, which is better